rest-api/dao/participant_counts_over_time_service.py

```diff
@@ -19,14 +19,16 @@ def get_filtered_results(self, start_date, end_date, filters, stratification='EN
     :return: Filtered, stratified results by date
     """
 
-    filters_sql = self.get_facets_sql(filters)
+    # Filters for participant_summary (ps) and participant (p) table
+    filters_sql_ps = self.get_facets_sql(filters)
+    filters_sql_p = self.get_facets_sql(filters, table_prefix='p')
```

Comment:
I find this a little confusing... what fields do we filter on for p vs. ps? Shouldn't we just have one method call that gets the facets SQL for both?